roles/image_builder/vars/main.yml

```diff
@@ -11,3 +11,40 @@ microshift_image_compose_customizations:
     password: "openshift"
     key: "{{ pubkey_file }}"
     groups: '["users", "wheel"]'
+
+microshift_kickstart_post:
+  - "{{ lookup('ansible.builtin.template', '../templates/firewall_options.j2') }}"
+  - "{{ lookup('ansible.builtin.template', '../templates/lvms_setup.j2') }}"
+  - "{{ microshift_image_ovn_options_template | default(omit) }}"
+  - "{{ microshift_image_crio_proxy_template | default(omit) }}"
+microshift_kickstart_options:
```

Comment:
These can be broken up into its own separate kickstart vars instead of overriding. This work can be done in infra.osbuild or possibly a separate kickstart collection